### ISSUE TITLE
V3, V4 support

### ISSUE THREAD

**Issue by andrejj** [GrappigPanda/Paseto#76]
Not an issue but a question.

A PHP app communicating with our Elixir app wants to upgrade to paseto V3.
Due to some dependency updates they need to update their [PHP paseto package](https://github.com/paragonie/paseto) which includes compatible dependencies but is dropping support for V1 and V2.

Are there any intentions to bring V3 and V4 support into this package?

**Comment by Ianleeclark** [comment]
Yes it is planned, but at the Moment im not sure when i will get to it. PRs are welcome

-------- Original-Nachricht --------
Am 10. Okt. 2022, 16:07, schrieb Andrej Jančič :

**Comment by Nezteb** [comment]
Where is the best place to start if looking to contribute to V3/V4 implementations?

**Comment by joeljuca** [comment]
> _Where is the best place to start if looking to contribute to V3/V4 implementations?_

I'd say to start reading Elixir library guidelines, then reading Paseto docs itself to implement it. One can never be 100% ready to start, but I believe you could get support from the community if you'd share your progress here and/or on Elixir Forum.
